docs/specifications/tests/Zone-TP/zone10.md

```diff
@@ -53,8 +59,8 @@ In other cases the outcome of this Test Case is "pass".
 Message                       | Default severity level
 :-----------------------------|:-----------------------------------
 MULTIPLE_SOA                  | ERROR
-NO_RESPONSE                   | WARNING
-NO_SOA_IN_RESPONSE            | WARNING
+NO_RESPONSE                   | DEBUG
+NO_SOA_IN_RESPONSE            | DEBUG
 ONE_SOA                       | INFO
 WRONG_SOA                     | ERROR
```

Comment:
Shouldn't WRONG_SOA also be demoted to DEBUG?

Reply:
Yes. Will fix.